src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -832,6 +832,7 @@ private static int IndexOfNonWhiteSpaceCharacter(ITextSnapshotLine line)
                         {
                             FinishExecute(executionResult.IsSuccessful);
                         }
+                        _window.SubmissionCounter++;
```

Comment:
Will this count reset or clear or history command if the user used the UI to execute those?